tests/test.py

```diff
@@ -240,8 +320,6 @@ def run_terraform(*, image: str, final: bool = False):
     invalid_volumes = {invalid_test_dir: {"bind": working_dir, "mode": "rw"}}
     checkov_test_dir = TESTS_PATH.joinpath("terraform/tool/checkov")
     checkov_volumes = {checkov_test_dir: {"bind": working_dir, "mode": "rw"}}
-    kics_config_dir = TESTS_PATH.joinpath("terraform/tool/kics")
```

Comment:
This and the below removal of `kics`-related items are because `kics` is not in the `terraform` images.